coordinator/agent/agent.go

```diff
@@ -11,6 +11,8 @@ import (
 
 	"github.com/mattermost/mattermost-load-test-ng/api"
 	"github.com/mattermost/mattermost-load-test-ng/loadtest"
+	"github.com/mattermost/mattermost-load-test-ng/loadtest/control/simplecontroller"
+	"github.com/mattermost/mattermost-load-test-ng/loadtest/control/simulcontroller"
```

Comment:
Don't really like this kind of coupling here but I guess it's unavoidable until we take a final decision on how to properly handle configurations.

Reply:
Added [here](https://mattermost.atlassian.net/browse/MM-23908) to track this issue.